### PR TITLE
Add screenshot, closes #7

### LINKED ISSUE #7: Add screenshot
screenshot

### PR DESCRIPTION
**Pull request by probonopd**

### CODE REVIEW

**Comment by TheAssassin** [comment]
The screenshot file should be committed, I don't want to rely on GitHub's weird user storage.

I'd prefer a text-only solution, but I guess that's not possible while retaining the colors, is it?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # appimagelint
 
-appimagelint is a tool to check AppImages for common issues.
+`appimagelint` is a tool to check [AppImage](https://appimage.org/) files for common issues.
```
Comment (TheAssassin) [inline README.md]: I don't like the app's name monospaced, it's not like the app name and the command you call differ at all.

**Review by TheAssassin — COMMENTED** [review]
Thanks, just got a minor change request.

**Review by TheAssassin — CHANGES_REQUESTED** [review]
Screenshot should be committed, relying on some external storage is insecure.

**Comment by probonopd** [comment]
Screenshot added to repository, please merge so that we get the correct URL, I can then edit the URLs in the README.md and AppStream metainfo file.

**Comment by TheAssassin** [comment]
URL will be https://github.com/TheAssassin/appimagelint/blob/944f85f74ede650a86ce01a18217d8834e2b3bb1/resources/screenshot.png, even after merge.

**Review by TheAssassin — APPROVED** [review]

**Comment by TheAssassin** [comment]
Thanks.

**Pull request merged**